modules/ossm-migrating-workloads-using-the-istio-injection-label.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-injection-label_{context}"]
+= Migrating workloads using the istio injection label
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately, and in any order. For more information, see "Migrating gateways".
+====
+
+.Procedure
+
+. Find the current `IstioRevision` for your {SMProduct} 3.0 control plane by running the following command:
++
+[source,terminal]
+----
+$ oc get istios
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME             REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+ossm-3           1           1       0        ossm-3-v1-24-3    Healthy   v1.24.3   30s
+----
+
+. Copy the "ACTIVE REVISION" to use as your `istio.io/rev` label in the next step.
++
+[NOTE]
+====
+The naming format of your revisions depends on which upgrade strategy you choose for your `Istio` instance.
+====
```

Comment:
```suggestion
----
+
Copy the `ACTIVE REVISION` value to use as your `istio.io/rev` label in the next step.
+
[NOTE]
====
The naming format of your revisions depends on which upgrade strategy you choose for your `Istio` instance.
====
```

This copy instruction should be put right under the example output instead of a separate step for clarity.

Reply:
Done